kits/arm/ex_impedance_control.py

```diff
@@ -19,6 +19,8 @@
 m = create_mobile_io(lookup, phone_family, phone_name)
 if m is None:
     raise RuntimeError("Could not find Mobile IO device")
+
+m.send_layout('./layouts/ex_impedance_control.json')
 m.set_button_mode(1, 'momentary')
```

Comment:
Again, here and other places -- once we move to sending the layout file, we should set the button mode with it as well...